cuml/coordinate_descent_demo.ipynb

```diff
@@ -17,35 +17,11 @@
    "source": [
     "The model can take array-like objects, either in host as NumPy arrays or in device (as Numba or cuda_array_interface-compliant), as well  as cuDF DataFrames. \n",
     "\n",
-    "For information about cuDF, refer to the [cuDF documentation](https://rapidsai.github.io/projects/cudf/en/latest/) \n",
+    "For information about cuDF, refer to the [cuDF documentation](https://rapidsai.github.io/projects/cudf/en/latest/).\n",
     "\n",
-    "For information about cuML's lasso implementation: https://rapidsai.github.io/projects/cuml/en/latest/api.html#lasso-regression\n",
+    "For information about cuML's lasso implementation: https://rapidsai.github.io/projects/cuml/en/0.10.0/api.html#cuml.Lasso.\n",
     "\n",
-    "For information about cuML's elastic net implementation: https://rapidsai.github.io/projects/cuml/en/latest/api.html#elasticnet-regression"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import os\n",
-    "\n",
-    "import numpy as np\n",
-    "\n",
-    "import pandas as pd\n",
-    "import cudf as gd\n",
-    "\n",
-    "from sklearn.datasets import make_regression\n",
-    "from sklearn.metrics import mean_squared_error\n",
-    "from sklearn.model_selection import train_test_split\n",
-    "\n",
-    "from cuml.linear_model import Lasso as cuLasso\n",
-    "from sklearn.linear_model import Lasso as skLasso\n",
-    "\n",
-    "from cuml.linear_model import ElasticNet as cuElasticNet\n",
-    "from sklearn.linear_model import ElasticNet as skElasticNet"
+    "For information about cuML's elastic net implementation: https://rapidsai.github.io/projects/cuml/en/0.10.0/api.html#cuml.ElasticNet."
```

Comment:
I think it will be easier for the user to compare the sklearn and cuml imports if we keep all the import statements in the first cell.

Reply:
Hi @salonijain27,

I am not sure about it.

The closer the imports are to where they are used, the easier is for the reader to understand its usage.

Keeping all of them at the beginning makes that cell subject to contain unused imports (that already happened in the current notebooks version).

cuML and SKL cells are quite close to each other, which makes comparing methods between them quite easy to spot.

Might I ask you to please reconsider to keep the imports location as I suggested?

Thanks!

Reply:
Fixed with commit https://github.com/rapidsai/notebooks/pull/271/commits/9c4dcbfe00c493f83751a42f7ae5f1a6c700a651